Library/.rubocop.yml

```diff
@@ -353,7 +353,7 @@ Layout/LineLength:
       ' name "',
       ' pkg "',
       ' pkgutil: "',
-      '    sha256 "',
+      '    sha256 ',
```

Comment:
Not blocking

```suggestion
      '    sha256 cellar: ',
```

Reply:
No. There might be no cellar when it is the default cellar.

Reply:
Will it still exceed the line length requirement in that case?

Reply:
In that case no. But the check is not checking for too short lines, only for too long lines 😂

Reply:
Yes, that's why the lines with no cellar don't need to be exempted from the line length check, I think. Anyway, nbd

Reply:
Ha I see, yeah, makes sense. Let me fix this.